docs/testing.rst

```diff
@@ -0,0 +1,60 @@
+=======
+Testing best practices
+=======
+
+Using setUpClass
+================
+
+Doing a lot of work in the ``setUp`` call of a test class means that it will be run on every test. This
```

Comment:
There's also package and module. http://pythontesting.net/framework/nose/nose-fixture-reference/#package